index.html

```diff
@@ -12,11 +12,16 @@
 </head>
 <body>
   <header>
+    <div class="top_menu">
+      <a href="#"><img src="./assets/facebook.png" alt="facebook"></a>
+      <a href="#"><img src="./assets/twitter.png" alt="twitter"></a>
+      <span class="span">English</span>
+    </div>
```

Comment:
- [ ] Kindly add all missing components of the dark navbar so that it would look like the original 👍 
- Your design:
![Annotation 2022-06-24 012727](https://user-images.githubusercontent.com/85113567/175439960-cff70981-fb0d-4ce8-98d0-9cb08147a149.jpg)
- Required design:
![Annotation 2022-06-24 012750](https://user-images.githubusercontent.com/85113567/175439985-723a8749-2618-4e0f-82df-dbba4605ec2f.jpg)